Dockerfile.rhel7

```diff
@@ -1,14 +0,0 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.14 AS builder
```

Comment:
@jluhrsen There is a different here between Dockerfile and this - in Dockerfile its: 
```
FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
```

Reply:
so I think we need this to fix it back up:
https://github.com/openshift/release/pull/39281